docs/tutorials/lerobot/replay.rst

```diff
@@ -57,6 +57,13 @@ Now try to replay the first recorded episode on your robot:
         --control.repo_id=${HF_USER}/trossen_ai_solo_test \
         --control.episode=0 \
 
+.. note::
+
+    The leader arms will be disabled in replay mode.
+
+    This change was introduced in the latest update to allow users to replay episodes or run inference
```

Comment:
Link the specific changelog update - the "latest" update will change on the next update